docs/releases/ibexa_dxp_v3.2.md

```diff
@@ -68,15 +68,15 @@ and [GraphQL](https://doc.ibexa.co/en/latest/api/graphql/#jwt-authentication).
 
 See [JWT authentication](https://doc.ibexa.co/en/latest/guide/security/#jwt-authentication) to learn how to configure this authentication method.
 
-### Searching in Ibexa Commerce with Elasticsearch
+### Searching in Ibexa Commerce with Elasticsearch [[% include 'snippets/commerce_badge.md' %]]
```

Comment:
In v3.2 Elastic was part of https://github.com/ezsystems/ezplatform-ee/blob/v3.2.8/composer.json#L36

Reply:
True, you're right.